CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ Learn the tools, talk to us about what you want to change, and open a small PR.
 Awesome! You have the basics of open-source software development (if not check above), but not much modelling experience.
 
 First step is to start thinking like a modeller. To understand the fine details about our library and contribute meaningfully, get some modelling experience:
-- Go though our [Introductory Tutorial](https://mesa.readthedocs.io/latest/tutorials/intro_tutorial.html) and [Visualization Tutorial](https://mesa.readthedocs.io/latest/tutorials/visualization_tutorial.html). While going through them, dive into the source code to really see what everything does.
+- Go though our [Introductory Tutorial](https://mesa.readthedocs.io/latest/tutorials/0_first_model.html) and [Visualization Tutorial](https://mesa.readthedocs.io/latest/tutorials/4_visualization_basic.html). While going through them, dive into the source code to really see what everything does.
```

Comment:
Apologies rereading these could we just just it to something like 

Go though our series of introductory tutorials at [Getting Started](https://mesa.readthedocs.io/latest/getting_started.html). While going through them, dive into the source code to really see what everything does.